src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/AssessmentApiService.kt

```diff
@@ -19,7 +19,8 @@ class AssessmentApiService(
   fun getRecentAssessment(crn: String): OffenderAssessment? =
     assessmentApiClient.getAssessmentSummaries(crn)
       .filter {
-        it.voided == null &&
+        it.assessmentStatus == "COMPLETE" &&
+          it.voided == null &&
           it.completed != null &&
```

Comment:
Do we still need thic check if assessmentStatus == "COMPLETE"?

Reply:
I'm not sure, for me its belts and braces, we need to check it for null on the 55 week check anyway.

Reply:
I'll combine the checks.